### PR TITLE
French localization

### PR DESCRIPTION
**Pull request by pavelzw**
This also adds a localized string for `"Dark mode"`

### CODE REVIEW

**Comment by pavelzw** [comment]
@bagaswastu Maybe you can consider squashing the pull requests instead of merging it. That way the git tree doesn't look so messy.

**Comment by bgwastu** [comment]
> @bagaswastu Maybe you can consider squashing the pull requests instead of merging it. That way the git tree doesn't look so messy.

Okay, I'll try

File: lib/main.dart [hunk 1]
```diff
@@ -25,8 +25,9 @@ class App extends StatelessWidget {
       enableLog: true,
       home: SplashView(),
       supportedLocales: [
-        Locale('en', 'US'),
-        Locale('de', 'DE'),
+        Locale('en', ''),
+        Locale('de', ''),
+        Locale('fr', ''),
```
Comment (bgwastu) [inline lib/main.dart:30]: Is it okay not to include the country code?

Comment (pavelzw) [reply lib/main.dart:30]: I think by not including the country code you make the language available for all German/English/French speaking countries.

**Pull request merged**